.gitignore

```diff
@@ -37,5 +37,8 @@ codeception.yml
 php.ini
 wp-config.php
 
+#PHP Unit
+.phpunit.result.cache
+
 # IDE files
 .vscode/*
```

Comment:
Same thing about the end of line here.

Reply:
Made the changes in [07d3fe4](https://github.com/woocommerce/facebook-for-woocommerce/pull/2624/commits/07d3fe4f5ddfaba0b99825ab4799728177f812be) commit.